service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseManagementOrderGenerationServiceTest.java

```diff
@@ -0,0 +1,287 @@
+package uk.gov.hmcts.reform.fpl.service;
```

Comment:
Let's ask Raghu/Dough to review it :D

Reply:
yep, I am going to move it to QA for a sanity check